crates/cranelift/src/lib.rs

```diff
@@ -149,17 +133,15 @@ fn wasm_call_signature(isa: &dyn TargetIsa, wasm_func_ty: &WasmFuncType) -> ir::
         // Cranelift's ABI implementation generates unwinding directives
         // about pointer authentication usage, so we can't just use
         // `CallConv::Fast`.
-        CallConv::WasmtimeAppleAarch64
-    } else if isa.triple().architecture == Architecture::S390x {
-        // On S390x we need a Wasmtime calling convention to ensure
-        // we're using little-endian vector lane order.
-        wasmtime_call_conv(isa)
+        CallConv::AppleAarch64
```

Comment:
Do we always need something special here? Because I'd like this function to unconditionally use `CallConv::Tail` soon, but I don't understand the details of this apple aarch64 stuff.